README.md

```diff
@@ -42,7 +42,9 @@ program, and cannot run all programs:
 * Miri runs the program as a platform-independent interpreter, so the program
   has no access to most platform-specific APIs or FFI. A few APIs have been
   implemented (such as printing to stdout) but most have not: for example, Miri
-  currently does not support concurrency, or SIMD, or networking.
+  currently does not support SIMD or networking.
+* Miri currently does not check for data-races and other concurrency related
```

Comment:
*most other

It does check for deadlocks.

Reply:
It checks for deadlocks in sequential programs. I don't think we have any check that works in the presence of concurrency.

Reply:
When all threads are waiting it reports a deadlock:

https://github.com/rust-lang/miri/pull/1284/files#diff-662c0b0aac0e20c3e14130dbda316626R191

Reply:
But what if two threads block on each other, and a third thread keeps on running fine?

Reply:
Then it will keep running. At least it detects some multi threading deadlocks.

Reply:
TBH that is such a weak deadlock checker I'd rather not advertise it.^^

Reply:
I suggested "Miri currently does not check for data-races and **most** other concurrency related issues" (bold is my addition). That doesn't advertise the weak deadlock checker, while still being more correct as **some** concurrency related issues are detected.

Reply:
No strong opinion here, @vakaras resolve as you see fit as long as we don't claim that we find all deadlocks.